test/parallel/test-buffer-bigint64.js

```diff
@@ -37,7 +37,11 @@ const buf = Buffer.allocUnsafe(8);
   assert.throws(function() {
     const val = 0x10000000000000000n;
     buf['writeBigUInt64' + endianness](val, 0);
-  }, RangeError);
+  }, {
+    code: 'ERR_OUT_OF_RANGE',
+    message: 'The value of "value" is out of range. It must be ' +
+             '>= 0n and < 2n ** 64n. Received 18,446,744,073,709,551,616n'
```

Comment:
`18,446,744,073,709,551,616` is locale specific and going to fail in certain locales if Node.js is built against full-icu/linked to a system ICU.

Reply:
nvm, only numbers are formatted, not bigints.

Reply:
It might actually still be an issue. In Germany we use dots instead of commas to separate thousands. I'll just run `toLocaleString`. That should be safe.

Or should I use the manual implementation for numbers as well?